components/Map/Map.tsx

```diff
@@ -98,6 +98,38 @@ const getColor = (
   return colors[5]
 }
 
+const getColorFromDataPoint = (dataPoint: number | string): string => {
```

Comment:
This gets the job done, but would mean we have two independent definitions of the boundaries. I see there's a "boundaries" array as well. Could that be something to use?